src/MapSettings.hpp

```diff
@@ -97,6 +97,7 @@ struct TrailSettings {
   enum class Type: uint8_t {
     VARIO_1,
     VARIO_2,
+    VARIO_KOBO,
```

Comment:
Adding this in the middle breaks all existing configurations, doesn't it?

Reply:
Not sure to understand this (I am more a carpenteer than a coder) but I moved the line to the bottom of the list, I hope it works...

Reply:
Enum is indexed. So actually the index value is saved to the user's profile not the name. If you alter the list instead of appending it, it will change the relationship between index value and trail type.

Reply:
Ok, thanks for explaining, now it makes sense to me